scss/message/_message.scss

```diff
@@ -385,7 +385,7 @@
   text-align: center;
   margin: 26px 0px;
 
-  p {
+  .bubble {
```

Comment:
have you made sure that `p` does not have behavior, that this class does not overwrites?

Reply:
No but looks similiar enough